src/chocolatey.resources/helpers/functions/Get-WebHeaders.ps1

```diff
@@ -43,6 +43,9 @@ command line'.
 .PARAMETER IgnoredArguments
 Allows splatting with arguments that do not apply. Do not use directly.
 
+.PARAMETER Options
+Specify request headers. Please note that UserAgent function parameter, if specified, overwrite header present in this parameter.
+

```

Comment:
This should also specify the first version this was available in. I would venture a guess that it is 0.10.13. At least include that here (search other parameters for examples of this and keep it consistent.). Thanks!